modules/ROOT/pages/basic-authentication-simple-concept.adoc

```diff
@@ -5,29 +5,34 @@ endif::[]
 
 [width="100%", cols="5,15"]
 |===
->s| Policy Name | Basic Authentication: Simple
+>s| Policy Name | Basic Authentication - Simple
 >s|Summary      | Allows access based on the basic authorization mechanism, with a single user-password
 >s|Category |
->s| Since Mule Version | 4.0.0
+>s| Available from Mule version | 4.0.0
```

Comment:
I still don't like how this sounds, regardless of consistency with old texts.
This wording is an introductory fragment rather than a heading or row entry.
Let's change this going forward to "First Mule version available".

Reply:
Done! Will have to change it all other policies as well.